Marlin/src/feature/bedlevel/ubl/ubl_G29.cpp

```diff
@@ -771,6 +771,9 @@
           const float measured_z = probe_pt(rawx, rawy, stow_probe, g29_verbose_level); // TODO: Needs error handling
           z_values[location.x_index][location.y_index] = measured_z;
         }
+        MYSERIAL0.flush(); // G29 P2's take a long time to complete.   PronterFace can
```

Comment:
This really should be `SERIAL_FLUSH()` since the previous `SERIAL_` commands could be writing to both serial ports.